Washington-County/COUNTY-DRUPAL
# WCOR-1246: Update system date formats.

## ODE XX

### Use/Fix Case

Date formats need to be normalized and made more consistent with web content from our peer organizations. 

(This will not have a lot of current impact without datetime display on most content right now. It will largely be noticed as a change in admin views.)

### Implementation Details

- System default date formats updated. 
- Content admin view updated.

![image](https://user-images.githubusercontent.com/17837170/150199116-92759258-b0ee-48ca-8a3e-44a00c738201.png)

### Test Plan

- Review the date formats at /admin/config/regional/date-time.
- Check system view /admin/content and note that both date & time are displayed for Updated value. (This is a new format called Default short date & time).
- Check system view /admin/content/media and note that only short date is displayed for Updated value. (This is the original short format, but the timestamp has been removed.)
